### PR TITLE
Alexander drapun/main utest

### PR DESCRIPTION
**Pull request by SashaDrapun**
Tests of the following repositories: DirectionRepository, TaskRepository, TaskStateRepository, VTaskRepository, VUserTask, VUserTrack

### CODE REVIEW

File: DIMS-Core/DIMS-Core.Tests/DIMS-Core.Tests.csproj [hunk 1]
```diff
@@ -9,10 +9,12 @@
 
   <ItemGroup>
     <PackageReference Include="EntityFramework" Version="6.4.4" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="5.0.0-rc.1.20451.13" />
     <PackageReference Include="Moq" Version="4.14.5" />
     <PackageReference Include="nunit" Version="3.12.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.15.1" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
+    <PackageReference Include="xunit.extensibility.core" Version="2.4.1" />
```
Comment (leonidbogin) [inline DIMS-Core/DIMS-Core.Tests/DIMS-Core.Tests.csproj:17]: Two similar libraries nunit and xunit are included.
I think only Nunit is necessary.
image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs [hunk 2]
```diff
@@ -0,0 +1,103 @@
+﻿using DIMS_Core.DataAccessLayer.Entities;
+using DIMS_Core.DataAccessLayer.Repositories;
+using DIMS_Core.Tests.Infrastructure;
+using NUnit.Framework;
+using System.Linq;
+using System.Linq.Dynamic.Core;
+using System.Threading.Tasks;
+using Xunit;
```
Comment (leonidbogin) [inline DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs:8]: Also included are two similar libraries nunit and xunit.

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/TaskRepositoryTests.cs [hunk 3]
```diff
@@ -0,0 +1,107 @@
+﻿using DIMS_Core.DataAccessLayer.Entities;
+using DIMS_Core.DataAccessLayer.Repositories;
+using DIMS_Core.Tests.Infrastructure;
+using NUnit.Framework;
+using System.Linq;
+using System.Linq.Dynamic.Core;
+using System.Threading.Tasks;
+using Xunit;
```
Comment (leonidbogin) [inline DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/TaskRepositoryTests.cs:8]: Also included are two similar libraries nunit and xunit.

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/TaskStateRepositoryTest.cs [hunk 4]
```diff
@@ -0,0 +1,102 @@
+﻿using DIMS_Core.DataAccessLayer.Entities;
+using DIMS_Core.DataAccessLayer.Repositories;
+using DIMS_Core.Tests.Infrastructure;
+using NUnit.Framework;
+using System.Linq;
+using System.Linq.Dynamic.Core;
+using System.Threading.Tasks;
+using Xunit;
```
Comment (leonidbogin) [inline DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/TaskStateRepositoryTest.cs:8]: Also included are two similar libraries nunit and xunit.

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VTaskRepositoryTests.cs [hunk 5]
```diff
@@ -0,0 +1,56 @@
+﻿using DIMS_Core.DataAccessLayer.Repositories;
+using DIMS_Core.Tests.Infrastructure;
+using NUnit.Framework;
+using System.Linq;
+using System.Linq.Dynamic.Core;
+using Xunit;
```
Comment (leonidbogin) [inline DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VTaskRepositoryTests.cs:6]: Also included are two similar libraries nunit and xunit.

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTaskRepositoryTest.cs [hunk 6]
```diff
@@ -0,0 +1,57 @@
+﻿using DIMS_Core.DataAccessLayer.Repositories;
+using DIMS_Core.Tests.Infrastructure;
+using NUnit.Framework;
+using System.Linq;
+using System.Linq.Dynamic.Core;
+using Xunit;
```
Comment (leonidbogin) [inline DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTaskRepositoryTest.cs:6]: Also included are two similar libraries nunit and xunit.

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTrackRepositoryTest.cs [hunk 7]
```diff
@@ -0,0 +1,56 @@
+﻿using DIMS_Core.DataAccessLayer.Repositories;
+using DIMS_Core.Tests.Infrastructure;
+using NUnit.Framework;
+using System.Linq;
+using System.Linq.Dynamic.Core;
+using Xunit;
```
Comment (leonidbogin) [inline DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTrackRepositoryTest.cs:6]: Also included are two similar libraries nunit and xunit.

**Review by leonidbogin — COMMENTED** [review]
The code is similar to mine, so I can't find errors =)
Only found an unnecessary use of xunit.

File: DIMS-Core/DIMS-Core.Tests/DIMS-Core.Tests.csproj (hunk 1 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DIMS-Core.Tests.csproj:17]: XUnit dependency was removed
image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VTaskRepositoryTests.cs (hunk 5 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VTaskRepositoryTests.cs]: Removed unnecessary using of xunit from all test classes

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTaskRepositoryTest.cs (hunk 6 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTaskRepositoryTest.cs]: Removed unnecessary using of xunit from all test classes

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTrackRepositoryTest.cs (hunk 7 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTrackRepositoryTest.cs]: Removed unnecessary using of xunit from all test classes

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/TaskStateRepositoryTest.cs (hunk 4 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/TaskStateRepositoryTest.cs]: Removed unnecessary using of xunit from all test classes

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/TaskRepositoryTests.cs (hunk 3 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/TaskRepositoryTests.cs]: Removed unnecessary using of xunit from all test classes

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs (hunk 2 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs]: Removed unnecessary using of xunit from all test classes

File: DIMS-Core/DIMS-Core.DataAccessLayer/Context/DIMSCoreDatabaseContext.cs [hunk 8]
```diff
@@ -9,9 +9,15 @@ namespace DIMS_Core.DataAccessLayer.Context
     public partial class DIMSCoreDatabaseContext : DbContext
     {
         private readonly IConfiguration configuration;
+        private DbContextOptions<DIMSCoreDatabaseContext> options;
```
Comment (maximsan) [inline DIMS-Core/DIMS-Core.DataAccessLayer/Context/DIMSCoreDatabaseContext.cs:12]: Can it be read only too ?

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Infrastructure/RepositoryTestBase.cs [hunk 9]
```diff
@@ -0,0 +1,332 @@
...
+                    {
+                        StateId = 6,
+                        StateName = "Ready to development"
+                    }
+                };
+            context.TaskState.AddRange(taskStates);
+            #endregion
+
+            #region Seed VUserProfiles
+            var vUserProfiles = new VUserProfile[3];
```
Comment (maximsan) [inline DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Infrastructure/RepositoryTestBase.cs:195]: I believe you can use context.VUserProfile.Add(...) here directly in cycle and do not create extra variable, can't you ?

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+        {
+            // Arrange
+            int countDirections = 5;
+            var query = new DirectionRepository(context);
+
+            // Act
+            var result = query.GetAll();
+
+            //Assert
+            Assert.AreEqual(countDirections, result.Count());
```
Comment (maximsan) [inline DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs:26]:
```
var actualCount = list.Count();

// Assert
Assert.That(actualCount, Is.EqualTo(expectedCount));
```

It's more clear assertion from my perspective, try to use this NUnit notation everywhere
https://docs.nunit.org/articles/nunit/writing-tests/assertions/assertion-models/constraint.html

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/TaskRepositoryTests.cs [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+        public async TaskThread ShouldDelete()
+        {
+            // Arrange
+            int deleteId = 3;
+            var query = new TaskRepository(context);
+
+            // Act
+            await query.DeleteAsync(deleteId);                          //delete
+            context.SaveChanges();
+            TaskEntity result = await query.GetByIdAsync(deleteId);    //get
```
Comment (maximsan) [inline DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/TaskRepositoryTests.cs:100]: use _var_

**Review by maximsan — CHANGES_REQUESTED** [review]
Hello. I left some the same notes to you as to Leonid. BTW, please add some test for negative cases, all your test suites has only successful tests. For example if you trying to delete or to update not existed user, or if you are trying to get the user from DB by not existed id. Anyway good work. Thanks!

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+{
+    [TestFixture]
+    public class DirectionRepositoryTest : RepositoryTestBase
+    {
+        [Test]
+        public void ShouldReturnAll()
+        {
+            // Arrange
+            int countDirections = 5;
+            var query = new DirectionRepository(context);
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs:20]: Move all initializations of repository in constructor.

**Review by PushMeNow — CHANGES_REQUESTED** [review]
Hi.
Please look my comments through and make changes.

File: DIMS-Core/DIMS-Core.DataAccessLayer/Context/DIMSCoreDatabaseContext.cs (hunk 8 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.DataAccessLayer/Context/DIMSCoreDatabaseContext.cs:12]: image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Infrastructure/RepositoryTestBase.cs (hunk 9 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Infrastructure/RepositoryTestBase.cs:194]: image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs (hunk 10 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs]: image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+            string newName = "---";
+
+            var query = new DirectionRepository(context);
+            var updateDirection = await query.GetByIdAsync(updateId);  //get
+            updateDirection.Name = newName;
+
+            // Act
+            query.Update(updateDirection);                            //update
+            context.SaveChanges();
+            var result = await query.GetByIdAsync(updateId);            //get
```
[the remark this answers is not in the document]
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs]: image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/TaskRepositoryTests.cs (hunk 11 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/TaskRepositoryTests.cs]: image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs (hunk 12 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs]: image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VTaskRepositoryTests.cs [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+        public void GetByIdAsync_GetItemByExistingId_ItemFound()
+        {
+            int getId = 1;
+            const string returnTaskName = "Create database";
+            var result = query.GetByIdAsync(getId);
+            Assert.That(returnTaskName, Is.EqualTo(result.Result.Name));
+        }
+
+        [OneTimeTearDown]
+        public void CleanupQuery()
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VTaskRepositoryTests.cs:46]: It is not query

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTaskRepositoryTest.cs [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+{
+    [TestFixture]
+    public class VUserTaskRepositoryTest : RepositoryTestBase
+    {
+        private VUserTaskRepository query;
+
+        [OneTimeSetUp]
+        public void InitQuery()
+        {
+            query = new VUserTaskRepository(Context);
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTaskRepositoryTest.cs:17]: It is not query

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTaskRepositoryTest.cs [hunk 16]
```diff
@@ -0,0 +1,51 @@
...
+        public void GetByIdAsync_GetItemByExistingId_ItemFound()
+        {
+            int getId = 1;
+            const string returnTaskName = "Create database";
+            var result = query.GetByIdAsync(getId);
+            Assert.That(returnTaskName, Is.EqualTo(result.Result.TaskName));
+        }
+
+        [OneTimeTearDown]
+        public void CleanupQuery()
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTaskRepositoryTest.cs:46]: It is not query

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTrackRepositoryTest.cs [hunk 17]
```diff
@@ -0,0 +1,51 @@
...
+{
+    [TestFixture]
+    public class VUserTrackRepositoryTest : RepositoryTestBase
+    {
+        private VUserTrackRepository query;
+
+        [OneTimeSetUp]
+        public void InitQuery()
+        {
+            query = new VUserTrackRepository(Context);
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTrackRepositoryTest.cs:17]: It is not query

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTrackRepositoryTest.cs [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+        public void GetByIdAsync_GetItemByExistingId_ItemFound()
+        {
+            int getId = 1;
+            const string returnTrackNote = "Create table UserProfile";
+            var result = query.GetByIdAsync(getId);
+            Assert.That(returnTrackNote, Is.EqualTo(result.Result.TrackNote));
+        }
+
+        [OneTimeTearDown]
+        public void CleanupQuery()
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTrackRepositoryTest.cs:46]: It is not query

**Review by PushMeNow — CHANGES_REQUESTED** [review]
Hi man!
I did some comments. It is about naming so will not difficult to fix it.

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs [hunk 19]
```diff
@@ -0,0 +1,85 @@
...
+    [TestFixture]
+    public class DirectionRepositoryTest : RepositoryTestBase
+    {
+        private DirectionRepository query;
+
+        [OneTimeSetUp]
+        public void InitQuery()
+        {
+            query = new DirectionRepository(Context);
+        }
```
[the remark this answers is not in the document]
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs]: image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs [hunk 20]
```diff
@@ -0,0 +1,85 @@
...
+        {
+            int deleteId = 3;
+            await query.DeleteAsync(deleteId);                     
+            Context.SaveChanges();
+            Direction result = await query.GetByIdAsync(deleteId);    
+            Assert.That(result, Is.Null);
+        }
+
+        [OneTimeTearDown]
+        public void CleanupQuery()
```
[the remark this answers is not in the document]
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/DirectionRepositoryTest.cs]: image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/TaskRepositoryTests.cs [hunk 21]
```diff
@@ -0,0 +1,89 @@
...
+        {
+            int deleteId = 3;
+            await query.DeleteAsync(deleteId);                          
+            Context.SaveChanges();
+            var result = await query.GetByIdAsync(deleteId);   
+            Assert.That(result, Is.Null);
+        }
+
+        [OneTimeTearDown]
+        public void CleanupQuery()
```
[the remark this answers is not in the document]
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/TaskRepositoryTests.cs]: image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VTaskRepositoryTests.cs (hunk 14 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VTaskRepositoryTests.cs]: image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTaskRepositoryTest.cs (hunk 16 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTaskRepositoryTest.cs]: image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTaskRepositoryTest.cs (hunk 15 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTaskRepositoryTest.cs]: image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTrackRepositoryTest.cs (hunk 18 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTrackRepositoryTest.cs]: image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTrackRepositoryTest.cs (hunk 17 above)
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/VUserTrackRepositoryTest.cs]: image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/TaskRepositoryTests.cs [hunk 22]
```diff
@@ -0,0 +1,89 @@
...
+{
+    [TestFixture]
+    public class TaskRepositoryTests : RepositoryTestBase
+    {
+        private TaskRepository query;
+
+        [OneTimeSetUp]
+        public void InitQuery()
+        {
+            query = new TaskRepository(Context);
```
[the remark this answers is not in the document]
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Repositories/TaskRepositoryTests.cs]: image

File: DIMS-Core/DIMS-Core.DataAccessLayer/Context/DIMSCoreDatabaseContext.cs [hunk 23]
```diff
@@ -9,9 +10,15 @@ namespace DIMS_Core.DataAccessLayer.Context
     public partial class DIMSCoreDatabaseContext : DbContext
     {
         private readonly IConfiguration configuration;
```
[the remark this answers is not in the document]
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.DataAccessLayer/Context/DIMSCoreDatabaseContext.cs]: image

**Review by maximsan — COMMENTED** [review]
Hello! You still have some unresolved questions. Please, resolve them

File: DIMS-Core/DIMS-Core.BusinessLayer/Services/MemberService.cs [hunk 24]
```diff
@@ -26,7 +26,7 @@ public MemberService(IUnitOfWork unitOfWork, IMapper mapper)
 
         public async Task<IEnumerable<VUserProfileModel>> SearchAsync()
         {
-            var query = unitOfWork.VUserProfileRepository.Search();
+            var query = unitOfWork.VUserProfileRepository.GetAll();
```
[the remark this answers is not in the document]
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.BusinessLayer/Services/MemberService.cs]: image

File: DIMS-Core/DIMS-Core.BusinessLayer/Services/TaskService.cs [hunk 25]
```diff
@@ -26,7 +26,7 @@ public TaskService(IUnitOfWork unitOfWork, IMapper mapper)
 
         public async Task<IEnumerable<TaskModel>> SearchAsync()
         {
-            var query = unitOfWork.VTaskRepository.Search();
+            var query = unitOfWork.VTaskRepository.GetAll();
```
[the remark this answers is not in the document]
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.BusinessLayer/Services/TaskService.cs]: image

File: DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Infrastructure/RepositoryTestBase.cs [hunk 26]
```diff
@@ -0,0 +1,333 @@
...
+using System.Collections.Generic;
+using System.Text;
+using EntityTask = DIMS_Core.DataAccessLayer.Entities.Task;
+
+namespace DIMS_Core.Tests.Infrastructure
+{
+    [TestFixture]
+    public class RepositoryTestBase : IDisposable
+    {
+        protected DIMSCoreDatabaseContext Context;
```
[the remark this answers is not in the document]
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Infrastructure/RepositoryTestBase.cs]: image

[the remark this answers is not in the document]
Comment (SashaDrapun) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Infrastructure/RepositoryTestBase.cs]: It should be protected because we use this variable in inherited classes

[the remark this answers is not in the document]
Comment (maximsan) [reply DIMS-Core/DIMS-Core.Tests/DataAccessLayer/Infrastructure/RepositoryTestBase.cs]: Ok! But don't forget to name it correctly

**Review by PushMeNow — APPROVED** [review]
PR was checked and approved.

**Review by maximsan — APPROVED** [review]
Please be accurate with naming all the time. It's approved
